backend/utils.ts

```diff
@@ -27,31 +28,62 @@ export const getTaxJarConfig = async (
 };
 
 export const taxJarConfigIsValidToUse = (taxJarConfig: TaxJarConfig) => {
-    const defaultResponse = {
-      isValid: true,
-      status: 200,
-      message: ""
-    };
-  
-    if (!taxJarConfig.active) {
-      console.log("TaxJar is not active.");
-      return {
-        ...defaultResponse,
-        message: "TaxJar is not active.",
-        isValid: false,
-      };
-    } else if (!taxJarConfig.apiKey) {
-      console.log("TaxJar apiKey was not provided.");
-  
-      return {
-        message: "TaxJar apiKey was not provided.",
-        status: 404,
-        isValid: false,
-      };
-    }
-  
+  const defaultResponse = {
+    isValid: true,
+    status: 200,
+    message: "",
+  };
+
+  if (!taxJarConfig.active) {
+    console.log("TaxJar is not active.");
```

Comment:
```suggestion
    console.error("TaxJar is not active.");
```
When you use `warn` or `error` instead of `log` then this will have nice colors in error output in Vercel deployment logs and will be displayed in "Errors" tab 😉

Reply:
Yeah, but it is something natural. Like we don't have webhooks specific for channel, so you could have taxjar turned off for `channel-pln`

Reply:
ok, if that so then making this `.warn` would make too much noise